news/_posts/2017-04-18-releases-1Q17.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: news
+post-type: announcement
+permalink: /news/releases-1Q17
+title: "Three new releases and more GitHub goodness!"
+---
+
+We are happy to announce three new Scala releases: 2.11.11 concludes the 2.11 series, 2.12.2 brings bug fixes and small improvements, and 2.13.0-M1 sets the scene for modularization and improvements coming in the next major release next year. Many thanks to everyone who contributed!
+
+As part of our ongoing efforts at Lightbend to simplify contributing to Scala, we are thrilled to announce our move to GitHub for [bug reporting](https://github.com/scala/bug), as well as for [release notes](https://github.com/scala/scala/releases).
+
+For the highlights of these three releases, read on.
+
+<!-- break -->
+
+[Scala 2.11.11](https://github.com/scala/scala/releases/tag/v2.11.11) contains many smaller fixes and a few backports from 2.12. It also features improved error messages on missing dependencies; this improvement was developed by @jvican at the Scala Center under proposal SCP-009.
```

Comment:
Thank you for the mention!
  
Can we add at the end of this paragraph the following line: "The rest of implementation of SCP-009 was outsource to [a compiler plugin](https://github.com/scalacenter/classpath-shrinker)."?